caim_base/static/js/common.js

```diff
@@ -46,6 +46,29 @@ $('[data-timestamp]').each(function (idx, el) {
     el.text(fmt.format(dte));
 });
 
+
+$('.reply').click(function(button){
+    let buttonID = button.target.id;
+
+        // Reply form element
+    var replyForm = `
+    <form method="post" id="reply-form${buttonID}" class="reply-form" action="/comments/${buttonID}/reply">
+        {% csrf_token %}
```

Comment:
This is a django tag, and renders as {% csrf_token %} in the browser. As this is JS and not processed by django, you will need to get the CRSF token manually and append to the form OR include the form in the template. See the csrfHeaders code below.